### PR TITLE
fix: Homepage icon and text update 

### LINKED ISSUE #3815: [BUG][UX] The name Search for the Recipes tab is misleading
### First Check

- [X] This is not a feature request.
- [X] I added a very descriptive title to this issue (title field is above this).
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [X] This issue can be replicated on the demo site (https://demo.mealie.io/).

### What is the issue you are experiencing?

In the left sidebar the most important tab of the application, that is the one containing the recipes, is named "Search". I think that for improving the usability it should be named "Recipes".

I would also remote the string "Results" and the magnifier icon or replace it with the string "Recipes"
Rationale:
- All the other tabs have a name that reflects what they contain: Meal planner, Shopping lists, Timeline
- Search is a functionality of the page, but not the main focus. It would be like calling the mail inbox "Search" because we can search emails
- It always takes me that bit of extra cognitive effort to remember that "Search" is the page containing the recipes

### Steps to Reproduce

---

### Please provide relevant logs

---

### Mealie Version

v1.9.0, d96c36333b9cb9461c5dee96ae28b60d912b38fd

### Deployment

Docker (Linux)

### Additional Deployment Details

_No response_

### PR DESCRIPTION
**Pull request by tyme-dev**
## What type of PR is this?

_(REQUIRED)_

- bug
## What this PR does / why we need it:
_(REQUIRED)_

Updates text  / icon on sidebar and recipe explorer page to better fit homepage context.

## Changes

- Changes Icon on sidebar for homepage and on the homepage itself from mdiMagnify to mdisilverwareFork
- Update text on sidebar for homepage and on the homepage itself to say "Recipes" instead of "Search" or "Results" 

Before: 
image

After: 
image

## Which issue(s) this PR fixes:

_(REQUIRED)_
Fixes #3815

### CODE REVIEW

**Review by boc-the-git — CHANGES_REQUESTED** [review]
The only language file that should ever be modified directly is `en-US.json` - all other translations are handled via Crowdin.

The change, from the screenshots looks good! I just can't bring myself to review it when there's 43 files changed :)

**Comment by tyme-dev** [comment]
@boc-the-git oh okay, thank you for the clarification, i've slimmed it down to a more reasonable 4 file changes :)

File: frontend/components/Domain/Recipe/RecipeExplorerPage.vue [hunk 1]
```diff
@@ -126,8 +126,8 @@
       <RecipeCardSection
         v-if="state.ready"
         class="mt-n5"
-        :icon="$globals.icons.search"
-        :title="$tc('search.results')"
+        :icon="$globals.icons.silverwareFork"
+        :title="$tc('search.recipes')"
```
Comment (boc-the-git) [inline frontend/components/Domain/Recipe/RecipeExplorerPage.vue]:
```suggestion
        :title="$tc('general.recipes')"
```

File: frontend/components/Layout/DefaultLayout.vue [hunk 2]
```diff
@@ -157,9 +157,9 @@
 
       const topLinks = computed<SidebarLinks>(() => [
         {
-          icon: $globals.icons.search,
+          icon: $globals.icons.silverwareFork,
           to: `/g/${groupSlug.value}`,
-          title: i18n.tc("sidebar.search"),
+          title: i18n.tc("sidebar.recipes"),
```
Comment (boc-the-git) [inline frontend/components/Layout/DefaultLayout.vue]:
```suggestion
          title: i18n.tc("general.recipes"),
```

File: frontend/lang/messages/en-US.json [hunk 3]
```diff
@@ -630,7 +630,8 @@
     "search-hint": "Press '/'",
     "advanced": "Advanced",
     "auto-search": "Auto Search",
-    "no-results": "No results found"
+    "no-results": "No results found",
+    "recipes": "Recipes"
```
Comment (boc-the-git) [inline frontend/lang/messages/en-US.json]:
```suggestion
    "no-results": "No results found"
```

File: frontend/lang/messages/en-US.json [hunk 4]
```diff
@@ -836,7 +837,8 @@
     "parser": "Parser",
     "developer": "Developer",
     "cookbook": "Cookbook",
-    "create-cookbook": "Create a new cookbook"
+    "create-cookbook": "Create a new cookbook",
+    "recipes": "Recipes"
```
Comment (boc-the-git) [inline frontend/lang/messages/en-US.json]:
```suggestion
    "create-cookbook": "Create a new cookbook"
```

**Review by boc-the-git — CHANGES_REQUESTED** [review]
We already have general.recipes, so there's no real need to add a new "recipes" translation in. This has the added benefit of not needing people to go in and apply a new translation for it, it'll be in their language (assuming already translated) as soon as we merge this!

Otherwise, this change looks good and I'm happy to merge! Thanks :)

**Comment by tyme-dev** [comment]
@boc-the-git updated to use general.recipes

**Review by Kuchenpirat — COMMENTED** [review]
i would like to see a different icon used here. I think the choosen icon is a bit small in contrast to the others. 

I would either suggest mdiSilverwareForkKnife or just the primary mealie icon

Bellow are screenshots for both options. 

| Primary | ForkKnife |
| --- | --- |
| image | image |

**Comment by tyme-dev** [comment]
> i would like to see a different icon used here. I think the choosen icon is a bit small in contrast to the others.
> 
> I would either suggest mdiSilverwareForkKnife or just the primary mealie icon
> 
> Bellow are screenshots for both options.
> Primary 	ForkKnife
> image 	image

my preferment would be for mdiSilverwareForkKnife. @boc-the-git thoughts?

**Comment by boc-the-git** [comment]
> > i would like to see a different icon used here. I think the choosen icon is a bit small in contrast to the others.
> > I would either suggest mdiSilverwareForkKnife or just the primary mealie icon
> > Bellow are screenshots for both options.
> > Primary 	ForkKnife
> > image 	image
> 
> my preferment would be for mdiSilverwareForkKnife. @boc-the-git thoughts?

I'm fine with that too. I think _not_ using the logo is not a bad thing here.

**Comment by tyme-dev** [comment]
@boc-the-git updated to use mdiSilverwareForkKnife.

**Review by boc-the-git — APPROVED** [review]

**Comment by boc-the-git** [comment]
Thanks @tyme-dev !

**Comment by Kuchenpirat** [comment]
Thanks @tyme-dev for your work on this. Always nice to see new contributors submit a PR! 😊

**Comment by simgunz** [comment]
Thanks @tyme-dev. The UI is way more clean and understandable now.

**Pull request merged**